.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Verified Bug
+description: Report a confirmed bug in ODK Web Forms
+type: 'bug'
+assignees: ''
+---
+
+### Expected Behaviour
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+### Observed Behaviour
+
+<!-- A clear and concise description of what actually happened, including any error messages. -->
+
+### Reproduction Steps
+
+<!-- Provide a clear, step-by-step guide to reproduce the bug. -->
+
+1.
+2.
+3.
+
+### Screenshots or Videos
+
+<!-- Add any relevant screenshots, videos, or links to help illustrate the issue. -->
+
+### Environment
+
+- Browser version:
+- OS:
+- Web Forms version:
+
+### Additional Context
+
+<!-- Any other details, such as device info, network conditions, or related forum discussions? -->
```

Comment:
Maybe prompt the user to supply the form that demonstrates the bug? Could be the first step of the reproduction section?